x/auction/keeper/keeper.go

```diff
@@ -224,7 +231,11 @@ func (k Keeper) BeginAuction(ctx sdk.Context,
 	// determine how many usomm each sale token min unit costs
 	saleTokenPriceInUsomm := saleTokenMinUnitValue.Quo(usommMinUnitValue)
 
-	// TODO(bolten): we are not inspecting the MinimumSaleTokensUsdValue param and using it -- implement later?
+	// TODO(bolten): we should consider uncommenting this, but it would require fixing a bunch of unit tests
+	// saleTokenValueInUSD := saleTokenMinUnitValue.Mul(sdk.NewDecFromInt(startingTokensForSale.Amount))
+	// if saleTokenValueInUSD.LT(paramSet.MinimumSaleTokensUsdValue) {
+	// 	return errorsmod.Wrapf(types.ErrAuctionBelowMinimumUSDValue, "sale tokens USD value: %s, minimum USD value: %s", saleTokenValueInUSD, paramSet.MinimumSaleTokensUsdValue)
+	// }
```

Comment:
If I'm understanding correctly, without this check the auction will just end from inactivity because no one can submit a successful bid anymore? If that's all then it seems fine commented out.

Reply:
No, this is checking that when an auction is first started, the sale tokens are worth some minimum USD value. So it would stop one from beginning at all.